go/client/cmd_upak.go

```diff
@@ -44,11 +49,23 @@ func (c *cmdUPAK) Run() error {
 	if err != nil {
 		return err
 	}
-
 	if err = RegisterProtocolsWithContext(nil, c.G()); err != nil {
 		return err
 	}
 
+	if c.lite {
+		upakLite, err := userClient.GetUPAKLite(context.Background(), c.uid)
+		if err != nil {
+			return err
+		}
+		jsonOut, err := json.MarshalIndent(upakLite, "", "  ")
+		if err != nil {
+			return err
+		}
+		c.G().UI.GetTerminalUI().Output(string(jsonOut) + "\n")
```

Comment:
curious, do you know if this is escaped for malicious unicode strings? I don't think there's any arbitrary user input for upak lites so it's okay here regardless.

Reply:
dunno